docs/user_manual/orders.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: "11. Orders"
+section: 11
+---
+
+Once a pool is ready for sequencing, there maybe delay to batch multiple pools
+for a single flow cell and a pool maybe placed in multiple lanes. To facilitate
+this, MISO has the idea of an order that indicates how much sequencing needs to
+be performed and whether the required sequencing has been completed.
+
+Orders are based on a pool, sequencing platform, and the sequencing parameters.
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=section sub=sub section-title=section-title title="Active Orders List" %}
+
+The active orders page shows all the orders and the number of matching
+sequencing runs. From the "Tracking" list, click "Active Orders". There are
+tabs for each of the sequencing instrument platforms active in MISO.
+
+There are columns for the number of matching lanes in different states. The
+"Remaining" column indicates the number of "Requested" runs minus all the runs
+that have completed successfully.
+
+Orders are only shown on this page if the "Remaining" column is greater than
+zero.
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=section sub=sub section-title=section-title title="All Orders List" %}
+
+The all orders page is similar to the "Active Orders" list. From the "Tracking
```

Comment:
Can the first sentence of this and the Pending Orders List section below be updated to indicate how these pages are different from the Active Orders list? (I know you have it there in the third sentence, but the disjoint makes it harder to understand)